frontend/components/EventPage/SyncModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styled from 'styled-components'
+import {
+  CLUBS_GREY,
+  CLUBS_NAVY,
+  CLUBS_BLUE,
+  WHITE,
+} from '../../constants/colors'
+
+const ModalContainer = styled.div`
+  text-align: left;
+  padding: 40px;
+`
+
+const Title = styled.div`
+  color: ${CLUBS_GREY};
+  font-size: 25px;
+  font-weight: bold;
+`
+
+const Subtitle = styled.div`
+  margin: 15px 0;
+  color: ${CLUBS_NAVY};
+  font-size: 15px;
+  font-weight: bold;
+`
+
+const URLCell = styled.input`
+  flex: 1;
+  font-size: 15px;
+  outline: none;
+  border: solid 1px ${CLUBS_BLUE};
+  border-top-left-radius: 5px;
+  border-bottom-left-radius: 5px;
+  padding: 10px;
+  margin-left: 10px;
+`
+
+const URLCopyButton = styled.div`
+  font-size: 15px;
+  line-height: 15px;
+  background: ${CLUBS_BLUE};
+  color: ${WHITE};
+  align-self: stretch;
+  display: grid;
+  place-items: center;
+  padding: 10px;
+  border-top-right-radius: 5px;
+  border-bottom-right-radius: 5px;
+`
+
+const SyncModal = (props: {}): ReactElement => {
+  return (
+    <ModalContainer>
+      <Title>Sync To Calendar</Title>
+      <div
+        style={{
+          display: 'flex',
+          alignItems: 'center',
+          marginTop: '10px',
+        }}
+      >
+        <b>URL: </b>
+        <URLCell readOnly value={'https://pennclubs.com/abc'} />
+        <URLCopyButton>Copy</URLCopyButton>
```

Comment:
Instead of doing this yourself, you can use [Bulma](https://bulma.io/documentation/form/general/#form-addons), the current CSS library we're using.

This might save you some time with formatting, and will make it look a little more consistent with the rest of the site.

We also use [Feather Icons](https://feathericons.com/) for icons. You can use an icon with `<Icon name="x" />`. If the icon svg isn't already downloaded, you'll need to download it and put it in the right place.